app/talk/lib/paginator.cjsx

```diff
@@ -39,6 +39,8 @@ module?.exports = React.createClass
     firstAndLast: true
     pageSelector: true
     scrollOnChange: true
+    previousLabel: <span><i className="fa fa-long-arrow-left" /> Previous</span>
```

Comment:
These are just changes to allow some customization and integration points with the paginator.